usr/local/sbin/resticw

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Exit on error, unbound variable, pipe error
+set -euo pipefail
+
+DOC="A little wrapper over restic just to handle profiles and environment loading.
+
+  It loads the backup profile/environment in a subshell to avoid any credential leak (Note: Run it with sudo so it can load the environment).
+
+  Usage:
+    resticw [options] <restic_arguments>
+
+    The restic_arguments is just the regular unwrapped restic arguments, e.g. stats latest
+
+  Options:
+    -p --profile=<name>        Specify the profile to load or use default [default: default].
+
+  Examples:
+    sudo resticw --profile profileA snapshots
+    sudo resticw stats latest  # this will use the profile: default
+"
+
+# The following argument parser is generated with docopt.sh from the above docstring.
+# See https://github.com/andsens/docopt.sh. If the DOC is updated or new options are added, refresh the parser!
+
+# docopt parser below, refresh this parser with `docopt.sh resticw`
+# shellcheck disable=2016,1075
+docopt() { parse() { if ${DOCOPT_DOC_CHECK:-true}; then local doc_hash
```

Comment:
I think docopt looks cook, but it's sure overkill for this use case of having just one option. Just checking manually `$1`, or using the standard built-in `getopts` (see my template here https://github.com/erikw/templates/blob/master/bash/getopts.sh#L30-L38) would suffice.

But programming should also be fun, and trying out new techs is fun,... so let's give it a try :)

Reply:
Yeah, hehe. I liked to save myself the burden of writing that tough stuff. And it comes with nice benefits, like short and long options, `--help` and usage out-of-the-box :D  Also good way to try stuff.

Reply:
Also forces you to write good usage DOC messages, hehe

Reply:
I don't say no to good documentation!